AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1352,6 +1352,7 @@ quotev.com#$#body #res_area ::selection { background-color: #3297FD!important; }
 !
 !--- JS ----
 !
+outdoornews.co.kr#%#AG_onLoad(function() { setTimeout(function() { $('#article-view-content-div').off('copy'); }, 300); });
```

Comment:
We have to avoid using JS rules. Are there no more options? What about scriptlets? Please check
```
outdoornews.co.kr#%#//scriptlet('abort-current-inline-script', '$', 'copy')
```

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/91034/commits/d38e8cc9a8a01c085749ee987f8f9ff45c226666